README.md

```diff
@@ -776,6 +776,60 @@ wait_agent(3, question, ScKeynodes[QuestionStatus.QUESTION_FINISHED])
 question, is_successfully = execute_agent(**kwargs, wait_time=3)  # ScAddr(...), bool
 ```
 
+### Prepare, call and execute action
+
+Function that allow to create action and call later.
+
+Prepare action function creates action and adds arguments and concepts.
```

Comment:
Can you provide a case when this logic can be used? I didn't see any reason to create such construction in memory, and in a separate place call it

Reply:
Action queue. I need to create and return an action node, wait for completion, and then run